db/seeds.rb

```diff
@@ -128,7 +129,7 @@ def rugged_repo
   rescue => e
     #puts e.message
     #puts e.backtrace
-    puts "Error creating concept exercises for Track #{track_slug}: #{e}"
+    puts "Error seeding Track #{track_slug}: #{e}"
```

Comment:
This made it easier to find any issues with tracks as well as skipping over any issues without crashing